corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -9,14 +9,38 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
         template: _.template($("#query-view-item-template").html() || ""),
 
         templateContext: function () {
-            var imageUri = this.options.model.get('imageUri');
-            var audioUri = this.options.model.get('audioUri');
-            var appId = this.model.collection.appId;
+            var imageUri = this.options.model.get('imageUri'),
+                audioUri = this.options.model.get('audioUri'),
+                appId = this.model.collection.appId,
+                initialValue = this.options.model.get('value');
+
+            // Initial values are sent from formplayer as strings, but dropdowns expect an integer
+            if (initialValue && this.options.model.get('input') === "select1") {
+                initialValue = parseInt(initialValue);
+            }
+
             return {
                 imageUrl: imageUri ? FormplayerFrontend.getChannel().request('resourceMap', imageUri, appId) : "",
                 audioUrl: audioUri ? FormplayerFrontend.getChannel().request('resourceMap', audioUri, appId) : "",
+                value: initialValue,
             };
         },
+
+        ui: {
+            valueDropdown: 'select.query-field',
```

Comment:
I know it's not a common pattern in our repo, but should we use constants for these names? Or is this pretty common in js as well?

Reply:
For the selectors? We'll use variables/constants sometimes if they're reused, but not for one-offs.